app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    _, target, new_name = command.split()
```

Comment:
The command string is split without validation. Ensure that the command string contains exactly three parts to avoid `ValueError` when unpacking.